### PR TITLE
feat: Design tokens

### PR DESCRIPTION
**Pull request by castastrophe**
## Creating a design token base

* Link to discovery documentation (should contain all the planning and requirements for this work):
   - https://rally1.rallydev.com/#/detail/userstory/389177352936?fdp=true
* Link(s) to demo pages where this element can be viewed:

---

### What has changed and why
Summarize files edited as part of this MR along with a brief description of what was changed/why.

*

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1. 

#### Browser requirements
Your component should work in all of the following environments:

- [ ] Latest 2 versions of Edge
- [ ] Internet Explorer 11 (should be useable, not pixel perfect)
- [ ] Latest 2 versions of Firefox (one on Mac OS, one of Windows OS)
- [ ] Firefox 60.7.2 (or latest version for Red Hat Enterprise Linux distribution)
- [ ] Latest 2 versions of Chrome (one on Mac OS, one of Windows OS)
- [ ] Latest 2 versions of Safari
- [ ] Galaxy S9 Firefox
- [ ] iPhone X Safari
- [ ] iPad Pro Safari
- [ ] Pixel 3 Chrome

Your repository infrastructure updates should work for at least:
- [ ] Node v8.x
- [ ] NPM v7.x

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Did browser testing pass?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?
- [ ] Did you update the CHANGELOG.md file with a summary of this update?

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**

### CODE REVIEW

File: tokens/color/base.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
+{
```
Comment (starryeyez024) [inline tokens/color/base.json]: this is super freaking exciting!! Thanks for exploring this! 😀  🎉

Comment (starryeyez024) [reply tokens/color/base.json]: cc @dcaryll @coreyvickery

Comment (castastrophe) [reply tokens/color/base.json]: I used the hex that is defined in the system that but would it make the colors more globally useable if they were represented in rgb?

Comment (starryeyez024) [reply tokens/color/base.json]: i think hex is totally fine for now. if we have a good reason to try to convert them later we can

**Comment by castastrophe** [comment]
Issue #850

**Pull request closed without merging**